components/external-services-mock/internal/subscription/config.go

```diff
@@ -0,0 +1,40 @@
+package subscription
+
+import (
+	"fmt"
+	"strings"
+)
+
+const (
+	TenantPathParamValue = "tenant"
+	RegionPathParamValue = "eu-1"
```

Comment:
I'm not sure if we can remove `TenantPathParamValue` and `RegionPathParamValue` values because we use them to build tenant fetcher full regional url that looks like: `https://compass-gateway.kyma.local/tenants/v1/regional/{region}/callback/{tenantId}`. And on local setup tenant-fetcher component expect this url in order to handle subscription request. 
About provider onSubscription callback path - `/tenants/v1/regional/{region}/callback/{tenantId}` you are right, we could simply it but my idea was to be closer to the real setup, also as you mention this will require updating saas-registry instance from that we'll need to generating new key, update saas-registy secret for the cluster and validate the whole scenario.
I would like to left it like it's now if you are okay with this?